statusmonitor/uves/control.go

```diff
@@ -75,38 +108,38 @@ type ControlUVEStatus struct {
 	} `json:"NodeStatus"`
 
 	BgpRouterState struct {
-		NumDownServiceChains      [][]interface{}         `json:"num_down_service_chains"`
-		BgpRouterIPList           [][]interface{}         `json:"bgp_router_ip_list"`
-		NumUpXMPPPeer             [][]NumUpXMPPPeer       `json:"num_up_xmpp_peer"`
```

Comment:
If structure NumUpXMPPPeer is not used anymore please remove it.

Reply:
Done